examples/nlp/text_classification_tf_keras/adaptive.yaml

```diff
@@ -0,0 +1,21 @@
+description: text_classification_tf
+hyperparameters:
+  global_batch_size: 32
+  dense1: 5
+  embedding_dim:
+    type: int
+    minval: 16
+    maxval: 32
+records_per_epoch: 60000
+searcher:
+  name: adaptive_asha
+  metric: val_sparse_categorical_accuracy
+  smaller_is_better: false
+  max_length:
+    epochs: 5
+  max_trials: 10
+environment:
+  image:
+    gpu: determinedai/environments:cuda-11.0-pytorch-1.7-lightning-1.2-tf-2.4-gpu-0.12.0
+    cpu: determinedai/environments:py-3.7-pytorch-1.7-lightning-1.2-tf-2.4-cpu-baefbf7
```

Comment:
Why do you need a CPU image here? Also, for the gpu image is this not the default image we use -- just asking not 100% sure? Ditto for const.yaml and distributed.yaml.

Reply:
i thought users could customize which resources the tasks were run on? like specify CPU for tensorboard, GPU for tasks? 
as for why it's not the default image, it's because my model uses a TextVectorization import that's only in tensorflow 2.0+. the default image is tensorflow 1.0

Reply:
Ok yeah this makes sense. Didn't think about the Tensorboard CPU case.